server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1821,7 +1822,8 @@ private SplitComputations(Set<FileRef> inputFiles, SortedMap<Double,Key> midPoin
     }
   }
 
-  private AtomicReference<SplitComputations> lastSplitComputation = new AtomicReference<>();
+  private volatile SoftReference<SplitComputations> lastSplitComputation =
+      new SoftReference<>(null);
```

Comment:
the lock below should be `final` ?

Reply:
changed in [682832a](https://github.com/apache/accumulo/pull/3261/commits/682832a88b58f109b60aae38edd3ff215b7b6708)